### PR TITLE
45- Styling the modal page-Delnia

### PR DESCRIPTION
**Pull request by DelniaAlipour**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

##45

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/components/CustomAudioPlayer.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  align-items: center;
+  cursor: pointer;
+  border: none;
+  outline: none;
+  padding-right: 0;
+  fill: #CB3100;
+}
+
+.stop-rectangles {
+  width: 13.92px;
```
Comment (mferryRV) [inline client/src/components/CustomAudioPlayer.css]: Is there a reason these px numbers need to be so precise?

File: client/src/components/CustomAudioPlayer.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+
+import React, { useState, useRef } from "react";
+import "./CustomAudioPlayer.css";
+
+function CustomAudioPlayer({ audioUrl }) {
```
Comment (mferryRV) [inline client/src/components/CustomAudioPlayer.js:4]: Wow - didn't even know we needed this and you've already built it!

**Review by mferryRV — APPROVED** [review]
Exemplary :+1:

File: client/src/components/CustomAudioPlayer.css (hunk 1 above)
Comment (DelniaAlipour) [reply client/src/components/CustomAudioPlayer.css]: I copied from figma :)

**Pull request merged**